pages/common/rustup.md

```diff
@@ -27,3 +27,7 @@
 - Run cargo build with a certain toolchain:
 
 `rustup run {{toolchain_name}} cargo build`
+
+- Open the local rust documentation in your browser:
+
+`rustup doc`
```

Comment:
Does `rustup documentation` work?

Reply:
yes. might depend on the version.

Reply:
or do you mean written as 'rustup documentation'? No that doesn't work.

Reply:
Nope, the subcommand is `rustup doc`

Reply:
doc is still intuitive however documentation works better. 

See what overs think before adding it.

Reply:
ok then.

Reply:
> doc is still intuitive however documentation works better.
> 
> See what overs think before adding it.

I don't know what you mean. The subcommand `rustup documentation` does not exist.